### PR TITLE
ens section fixes

### LINKED ISSUE #1993: Profile: ENS unsernames fixes
Screen_Shot_2021-03-05_at_10_55_11

Dark image: https://www.figma.com/file/k2h3UyN2f5m4rYppxfOzFE/%F0%9F%8C%93-Dark-Mode%E2%8E%9CDesktop?node-id=4753%3A88

Screen_Shot_2021-03-05_at_11_03_19
ui

### PR DESCRIPTION
**Pull request by rinzlxr**
fixes #1993

- [x] added new image
- [x] delete titles on the screens
- [x] ensure 624 max width on both screen (and centred)
- [x] button to match dark mode (there arent any images for dark mode version of an arrow so i used `StatusRoundButton` similar to the login screen where its visible if the user has entered the correct input)

### CODE REVIEW

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 1]
```diff
@@ -65,132 +66,132 @@ Item {
...
-            source: "../../../../img/block-icon-white.svg"
-            width: 20
-            height: 20
-            anchors.horizontalCenter: parent.horizontalCenter
-            anchors.verticalCenter: parent.verticalCenter
+        StyledText {
+            id: sectionTitle
+            text: ""
+            anchors.left: parent.left
+            anchors.leftMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:79]:
```suggestion
            anchors.leftMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 2]
```diff
@@ -65,132 +66,132 @@ Item {
...
+            anchors.top: parent.top
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:81]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 3]
```diff
@@ -65,132 +66,132 @@ Item {
...
-            visible: ensStatus !== "taken"
-            text: {
-                if((ensStatus === "available" || ensStatus === "connected" || ensStatus === "connected-different-key")){
-                    return "✓"
-                } else {
-                    return "@"
+        Rectangle {
+            id: circleAt
+            anchors.top: sectionTitle.bottom
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:89]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 4]
```diff
@@ -65,132 +66,132 @@ Item {
...
+                width: 20
+                height: 20
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.verticalCenter: parent.verticalCenter
+            }
+
+            StyledText {
+                visible: ensStatus !== "taken"
+                text: {
+                    if((ensStatus === "available" || ensStatus === "connected" || ensStatus === "connected-different-key")){
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:110]: We should probably add those statuses in the Constants

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 5]
```diff
@@ -65,132 +66,132 @@ Item {
...
-        anchors.right: btnContinue.left
-        anchors.rightMargin: 24
-        Keys.onReleased: {
-            onKeyReleased(ensUsername.text);
-        }
+        Input {
+            id: ensUsername
+            placeholderText: !isStatus ? "vitalik94.domain.eth" : "vitalik94"
+            anchors.top: circleAt.bottom
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:129]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 6]
```diff
@@ -65,132 +66,132 @@ Item {
...
+            anchors.right: btnContinue.left
+            anchors.rightMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:131]:
```suggestion
            anchors.rightMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 7]
```diff
@@ -65,132 +66,132 @@ Item {
...
-        MouseArea {
-            cursorShape: Qt.PointingHandCursor
-            anchors.fill: parent
-            onClicked : {
+        StatusRoundButton {
+            id: btnContinue
+            width: 44
+            height: 44
+            anchors.top: circleAt.bottom
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:186]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 8]
```diff
@@ -210,62 +211,70 @@ Item {
                 }
             }
         }
-    }
 
+        Rectangle {
+            id: ensTypeRect
+            anchors.top: ensUsername.bottom
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:218]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 9]
```diff
@@ -210,62 +211,70 @@ Item {
...
-        border.color: Style.current.border
-        color: Style.current.background
-        radius: 50
-        height: 20
-        width: 350
+            Item {
+                anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.top: parent.top
+                anchors.topMargin: 8
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:230]:
```suggestion
                anchors.topMargin: Style.current.halfPadding
```

File: ui/app/AppLayouts/Profile/Sections/Ens/Search.qml [hunk 10]
```diff
@@ -210,62 +211,70 @@ Item {
...
-        text: validationMessage
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.top: ensTypeRect.bottom
-        anchors.topMargin: 24
+        StyledText {
+            id: validationResult
+            text: validationMessage
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.top: ensTypeRect.bottom
+            anchors.topMargin: 24
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/Ens/Search.qml:277]:
```suggestion
            anchors.topMargin: Style.current.bigPadding
```

**Review by jrainville — COMMENTED** [review]
Code looks good, just need to replace some numbers with the Theme ones

**Comment by rinzlxr** [comment]
i replaced common strings and numbers with constants available in `Constants.qml` , and added new ones as well

**Comment by iurimatias** [comment]
Screen Shot 2021-03-24 at 2 07 40 PM

**Pull request merged**